### DISCUSSION TITLE
Is the source code for the examples on the website published somewhere?

### DISCUSSION THREAD

**Discussion by laalsaas** [typst/typst#3926] [General]
On the website there are a lot of cool examples. I'd think that it would be ideal to add a link to the source code of each example, so that if someone thinks: „cool, i wanna do that in my document“, they can just see how it's done.

Personally, I'm specifically interested in the sine wave example [here](https://typst.app/assets/images/readyforscience.png), but i think it would generally be pretty handy.

**Comment by interdisziplina** [comment]
Diagrams are not implemented yet iirc, so this is likely just an image embedded into the document via the [image](https://typst.app/docs/reference/visualize/image/) function. I agree that once Typst has the full capability to render all of the examples by itself without any external tools it would be handy to add the source code.
